html/elements/input/color.json

```
@@ -36,10 +36,10 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": "10"
+                "version_added": "12.1"
               },
               "safari_ios": {
-                "version_added": false
+                 "version_added": "12.2"
```

I accidentally added a space here 🙈 
```suggestion
                "version_added": "12.2"
```